### ISSUE TITLE
nirvana api print fatal error invalid character '\x1b' looking for beginning of value

### ISSUE THREAD

**Issue by narrowizard** [caicloud/nirvana#335]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind feature

**What happened**:
nirvana print a fatal error when running `nirvana api`.
```shell
$ nirvana api --output="./api"
INFO  0611-14:39:33.220+08 api.go:87 | No packages are specified, defaults to pkg/apis
FATAL 0611-14:40:03.467+08 api.go:61 | invalid character '\x1b' looking for beginning of value
```
**What you expected to happen**:
the error is unexpected.
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:  
after some searching, i find the error is reported by `json.Unmarshal`.  
`nirvana api` generated a `nirvana-generated***` file and run it to get a json formated `api-definition` string, then output the string to standard output(cmd). and read the string from standard output(cmd), decode it to `&api.Definitions{}`.  
the error occur when my project output something to cmd.  

![std.out](https://user-images.githubusercontent.com/11383928/84354210-11aa5680-abf3-11ea-87c1-db0b30e98192.png)

![nirvana-generated****](https://user-images.githubusercontent.com/11383928/84353417-af9d2180-abf1-11ea-9bcf-2e1d1e17b487.png)
https://github.com/caicloud/nirvana/blob/bf6c5fc90fa4e45dbed1e0675d1ef9687fc73a8c/utils/builder/builder.go#L172-L183

**Comment by iawia002** [comment]
So you are logging something in the init function of descriptors or modifiers file?

**Comment by narrowizard** [comment]
not exactly, init function in other package which is imported by descriptors. @iawia002

**Comment by iawia002** [comment]
Got it, I'll fix this when I'm free (may take days)
